backend/src/controllers/checkoutController.js

```diff
@@ -1,15 +1,12 @@
 const Checkout = require('../database/model/checkout');
-const tracer = require('dd-trace').init();
-const axios = require('axios')
 
 
 class CheckoutController {
   Register(request, response) {
-    const span = tracer.scope().active()
     try{
       const { amount, type} = request.body;
       if (amount) {
-        const checkoutCreated = Checkout.create({ typerrrrr, amount });
+        const checkoutCreated = Checkout.create({ type, amount });
         response.status(200).json(checkoutCreated);
       } else{
         span.setTag('error.msg', "Valor de 'amount' chegou nulo");
```

Comment:
since you removed the tracer we need to get rid of this one too

Reply:
LGTM too, I did it now. tks